docs/SetupGuide_Dotnet.md

```diff
@@ -213,7 +213,9 @@ If the `{name}` specified in the `getproducts-namenull/{name}` URL is "null", th
 
 #### IAsyncEnumerable
 
-Using the `IAsyncEnumerable` binding generally requires that the `Run` function be `async`. It is also important to call `DisposeAsync` at the end of function execution to make sure all resources used by the enumerator are freed.
+Using the `IAsyncEnumerable` binding generally requires that the `Run` function be `async`.
+
+**Note:** It is important to call `DisposeAsync` at the end of function execution to make sure all resources used by the enumerator are freed. If `DisposeAsync` is not called, the SQL connection will remain open and result in resource leaks.
```

Comment:
@dzsquared Do you know of any docs we could link to about properly disposing SqlConnections? https://learn.microsoft.com/en-us/dotnet/api/microsoft.data.sqlclient.sqlconnection?view=sqlclient-dotnet-standard-5.1 has a tiny bit but nothing really specific about it, and it'd be nice to have some "official" guidance on why this is important.